Translated/UI/ui_mainmode.csv

```diff
@@ -6052,7 +6052,7 @@ SizeDot#0,"""."""
 SizePunctuation#0,"""、"""
 First_Combo#0,"""<c 60ccf0>コンボカウント開始！<c><br>採取や釣りを $(0)コンボで、フィーバー確定！<br>コンボ終了時に<green>経験値<c>獲得！"""
 First_Fever#0,"""<c f060ae>フィーバー開始！<c><br>一定時間、採取と釣りの<br><yellow>獲得数＆レア率アップ！<c>"""
-First_Point_End#0,"""<c 78f060>ねこそぎボーナス！<c><br>このスポットをすべてギャザリングしました！<br>コンボ数にボーナス<yellow>＋$(0)<c>！"""
+First_Point_End#0,"""<c 78f060>ねこそぎボーナス！<c><br>このスポットをすべてギャザリングしました！<br>コンボ数にボーナス<yellow>+$(0)<c>！"""
```

Comment:
Same here. Line isn't translated, so don't touch the punctuation.

Reply:
resolved